src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -303,13 +304,9 @@ protected override void Shutdown()
 
             await requestContext.SendResult(null);
 
-            // If no script is being launched, execute an empty script to
-            // cause the prompt string to be evaluated and displayed
-            if (string.IsNullOrEmpty(this.scriptToLaunch))
-            {
-                await this.editorSession.PowerShellContext.ExecuteScriptString(
-                    "", false, true);
-            }
```

Comment:
So what do we miss out on during an interactive session debug by not having the above code? An initial prompt string?